Pod/Classes/VOKCoreDataManager.h

```diff
@@ -302,7 +309,7 @@ typedef void(^VOKObjectIDsReturnBlock)(VOKArrayOfManagedObjectIDs *managedObject
  @param objectClass     Specifies the class to instantiate or fetch when importing data.
  @param completion      Fired on the main queue once the changes have been merged. It brings an NSArray of permanent NSManagedObjectIDs matching the objects deserialized from the import array.
  */
-+ (void)importArrayInBackground:(VOKArrayOfObjectDictionaries *)inputArray
+- (void)importArrayInBackground:(VOKArrayOfObjectDictionaries *)inputArray
```

Comment:
If I left the old methods in place, they could be tagged with the deprecation macro with a message pointing to the new methods.

I'd say maybe we add the instance methods to 3.x and mark the class methods as deprecated there, with the note that they'll be removed in 4.x?